src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -1273,9 +1273,8 @@ impl GraphEditorModel {
         let is_macos     = ARGS.platform.map(|p|p.is_macos()) == Some(true);
         let is_frameless = ARGS.frame == Some(false);
         let x_offset     = if is_macos && is_frameless { MACOS_TRAFFIC_LIGHTS_WIDTH }
-                           else                        { MACOS_TRAFFIC_LIGHTS_SIDE_OFFSET };
+                           else                        { 0.0 };
```

Comment:
I changed this because I don't see why we should have the offset if the traffic lights are not their. Please correct me if there is a reason for the offset. Also, someone with Mac OS should check if the new design still works there.

Reply:
This was added here because we want to have the same padding of breadcrumbs fron the left side of the window as the padding between left side of the window and traffic lights (in fact we want to use the same padding for top padding too). Please adjust it so these paddings are the same.